bftengine/src/bftengine/ReadOnlyReplica.hpp

```diff
@@ -41,7 +42,7 @@ class ReadOnlyReplica : public ReplicaForStateTransfer {
 
   template <typename T>
   void messageHandler(MessageBase* msg) {
-    if (validateMessage(msg) && !isCollectingState())
```

Comment:
Could you please explain that change (as above)?

Reply:
We want to allow client requests (because the replica is mostly on ST and we want to allow the latestPruneableBlock request even then), but still to deny checkpoint messages